scanpipe/pipes/d2d.py

```diff
@@ -1344,4 +1338,10 @@ def flag_deployed_from_resources_with_missing_license(project, doc_extensions=No
             status=flag.IGNORED_DOC_FILE
         )
 
-    scanned_from_files.update(status=flag.REQUIRES_REVIEW)
+    no_license_files = scanned_from_files.filter(Q(detected_license_expression=""))
+    unknown_license_files = scanned_from_files.filter(
+        Q(detected_license_expression__icontains="unknown")
+    )
```

Comment:
We should use the available QuerySet  `unknown_license()` methods.

Reply:
Used this now, thanks!